__tests__/app.test.js

```diff
@@ -179,3 +179,49 @@ describe("GET /api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  test("status:201 should return the comment", () => {
+    const newComment = {
+      username: "dav3rid",
+      body: "terrible",
+    };
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toEqual(
+          expect.objectContaining({
+            body: "terrible",
+            review_id: 1,
+            author: "dav3rid",
+            comment_id: expect.any(Number),
+            created_at: expect.any(String),
+            votes: 0,
+          })
+        );
+      });
+  });
+  test('status:400 Should return "missing required fields" if the request body of the comment is malformed or is missing required fields', () => {
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send({})
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toEqual("Missing required fields");
+      });
+  });
+  test('status: 400 Should return "Invalid input" if the body or username of the request body is not a string ', () => {
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send({
+        username: 1,
+        body: 1,
+      })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toEqual("Invalid input");
+      });
+  });
```

Comment:
Good start on the testing 👍 

You are missing tests for :

Username is in the valid format but is not found in the database {username: Rich } - 404 

Trying to post to an article which is not found in the database (Id = 5483) - 404

Trying to post to an invalid article Id /api/reviews/chicken/comments - 400

Everything else in the ticket looks good! Well done 👍